src/executables/server.c

```diff
@@ -351,7 +352,9 @@ main (int argc, char **argv)
 
 #if !defined(WINDOWS)
     hb_set_exec_path (executable_path);
+    css_set_exec_path (executable_path);
     hb_set_argv (argv);
+    css_set_argv (argv);
```

Comment:
```suggestion
    hb_set_argv (argv);
    
    css_set_exec_path (executable_path);
    css_set_argv (argv);
```

ha와는 연관이 없기 때문에 붙일 필요는 없을 것 같습니다.

Reply:
hb 함수 따로, css 함수 따로 호출되도록 나누었습니다.